test/sol5/networkHelper.js

```diff
@@ -127,6 +136,29 @@ module.exports.setupReserves = async function
     return result;
 }
 
+module.exports.setupNetwork = setupNetwork;
+async function setupNetwork
+    (network, networkProxyAddress, KNCAddress, DAOAddress, tokens, accounts, admin, operator){
+    await network.addOperator(operator, { from: admin });
+    //init matchingEngine, feeHandler
+    let matchingEngine = await MatchingEngine.new(admin);
+    await matchingEngine.setNetworkContract(network.address, { from: admin });
+    await matchingEngine.setFeePayingPerReserveType(true, true, true, false, true, true, { from: admin });
+    let feeHandler = await FeeHandler.new(DAOAddress, network.address, network.address, KNCAddress, burnBlockInterval);
+    await network.setContracts(feeHandler.address, matchingEngine.address, zeroAddress, { from: admin });
+    // set DAO contract
+    await network.setDAOContract(DAOAddress, { from: admin });
+    // point proxy to network
+    await network.addKyberProxy(networkProxyAddress, { from: admin });
+    //add and list pair for reserve
+    let result = await setupReserves(network, tokens, 1, 1, 0, 0, accounts, admin, operator);
```

Comment:
seems to me setup reserves should not be here.
 a tester should be aware to which reserves are added.
so please remove setupRserves and line that adds reserves to network

Reply:
change as request